<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gen_sinus_box</class>
 <widget class="QFrame" name="Gen_sinus_box">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Синус</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>begin_angle</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>count_cycles</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>vert_ampl</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>offset_begin_y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dsb_offset_begin_y"/>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="dsb_vert_ampl"/>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dsb_begin_angle"/>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="dsb_count_cycles"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_generate">
        <property name="text">
         <string>Генерировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
